functions/utilities.zsh

```diff
@@ -6,6 +6,19 @@
 # https://github.com/bhilburn/powerlevel9k
 ################################################################
 
+################################################################
+# Source autoload functions
+################################################################
+local autoload_path="$__P9K_DIRECTORY/functions/autoload"
+# test if we already autoloaded the functions
+if [[ ${fpath[(ie)$autoload_path]} -gt ${#fpath} ]]; then
```

Comment:
What does this line technically do? Is there another way of checking if a function is already autoloaded?

Reply:
Technically, it checks if the fpath already contains our autoload_path or not, and then adds it if required. This is code I got from the ZSH manual for autoload.